packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/createproject.md

```diff
@@ -1,154 +1,144 @@
 Launch Xcode and create a new iOS app project using SwiftUI for the interface and Swift for the language.
 
-> **Note:** The sample code is compatible with iOS 14 and later.
+> **Note:** The sample code uses features of SwiftUI available in iOS 14, and the async/await feature of iOS 15. Use the completion handler version of the async/await functions to add compatibility with iOS 14 and later.
```

Comment:
In this case I didn't use code font because that's the way Apple presents the term on their developer website. At the moment I haven't made this change.

Reply:
Makes good sense. Thanks!